packages/location-field/src/index.js

```diff
@@ -525,6 +553,17 @@ class LocationField extends Component {
       menuItems.push(currentLocationOption);
       itemIndex++;
     }
+    if (message) {
+      const messageItem = (
+        <Option
+          icon={<ExclamationCircle size={20} />}
+          key={optionKey++}
+          title={message}
+          disabled
+        />
+      );
+      menuItems.unshift(messageItem);
+    }
```

Comment:
Since this always shows up with a exclamation circle icon, perhaps it would be better to prefix this variable and name it `errorMessage` and `errorMessageItem` respectively.

Reply:
The "no results found" message isn't really an error, so I don't think we necessarily should be calling it an error message.